next-frontend/src/app/(wca)/competitions/competitionClient.tsx

```diff
@@ -46,15 +46,13 @@ interface CompetitionsListProps {
 export default function CompetitionsClient({
   competitions,
 }: CompetitionsListProps) {
-  const countries = Object.entries(countryCodeMapping).map(([code, name]) => ({
-    label: name,
-    value: code,
-  }));
-
   const { contains } = useFilter({ sensitivity: "base" });
 
   const { collection, filter } = useListCollection({
-    initialItems: countries,
+    initialItems: Object.entries(countries.byIso2).map(([code, country]) => ({
+      label: country.id,
```

Comment:
Is this a temporary placeholder kind of thing? Using the ID as display value would probably lead to curious camelCase artifacts like `UnitedStatesOfAmerica` or perhaps even just `USA`...

Reply:
yes, this is temporary until the i18n PR is merged